models/issue_watch.go

```diff
@@ -83,3 +86,25 @@ func removeIssueWatchersByRepoID(e Engine, userID int64, repoID int64) error {
 		Update(iw)
 	return err
 }
+
+// LoadWatchUsers return watching users
+func (iwl IssueWatchList) LoadWatchUsers() (users UserList, err error) {
+	return iwl.loadWatchUsers(x)
+}
+
+func (iwl IssueWatchList) loadWatchUsers(e Engine) (users UserList, err error) {
+	if len(iwl) == 0 {
+		return nil, nil
```

Comment:
It's safer `return []*User{}, nil`.

Reply:
I'll have a grep on this later on ...

Reply:
done + addend check if len(userIDs) == 0